### ISSUE TITLE
vmwarefusion boot2docker running docker 1.5.0

### ISSUE THREAD

**Issue by jayed** [docker/machine#1020]
The machine virtualbox driver is installing docker 1.6.0 while the vmwarefusion driver is installing 1.5.0.

                        ##        .
                  ## ## ##       ==
               ## ## ## ##      ===
           /""""""""""""""""\___/ ===
      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~ ~ /  ===- ~~~
           \______ o          __/
             \    \        __/
              \____\______/
 _                 _   ____     _            _
| |__   ___   ___ | |_|___ \ __| | ___   ___| | _____ _ __
| '_ \ / _ \ / _ \| __| __) / _` |/ _ \ / __| |/ / _ \ '__|
| |_) | (_) | (_) | |_ / __/ (_| | (_) | (__|   <  __/ |
|_.__/ \___/ \___/ \__|_____\__,_|\___/ \___|_|\_\___|_|
Boot2Docker version 1.5.0, build vmw-1.5.0 : 67fcb7d - Wed Feb 18 10:00:58 UTC 2015
Docker version 1.5.0, build a8a31ef

**Comment by jayed** [comment]
Jays-MacBook-Air-2:localvm jayed$ docker ps
FATA[0000] Error response from daemon: client and server don't have same version (client : 1.18, server: 1.17) 
Jays-MacBook-Air-2:localvm jayed$ docker --version
Docker version 1.6.0, build 4749651

**Comment by ehazlett** [comment]
@frapposelli any chance we could get an updated vmwarefusion ISO?

**Comment by frapposelli** [comment]
Well, we were supposed to get boot2docker 1.6.0 with vmtools in it so we could switch to the mainline boot2docker release, see last @tianon comment here https://github.com/boot2docker/boot2docker/pull/747

Not sure why this didn't work out.

**Comment by frapposelli** [comment]
If we can't work this out I will cut another custom iso, but I'd rather not :)

**Comment by frapposelli** [comment]
This PR will fix it and also add shared folders support https://github.com/docker/machine/pull/1034

**Comment by ehazlett** [comment]
Closing as #1034 is merged which updates to 1.6.0.  Thanks!
